src/Ryujinx.Ui.Common/Helper/ShortcutHelper.cs

```diff
@@ -52,7 +52,7 @@ private static void CreateShortcutLinux(string applicationFilePath, byte[] iconD
         [SupportedOSPlatform("macos")]
         private static void CreateShortcutMacos(string appFilePath, byte[] iconData, string desktopPath, string cleanedAppName)
         {
-            string basePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, AppDomain.CurrentDomain.FriendlyName);
+            string basePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Ryujinx");
             var plistFile = EmbeddedResources.ReadAllText("Ryujinx.Ui.Common/shortcut-template.plist");
             // Macos .App folder
             string contentFolderPath = Path.Combine(desktopPath, cleanedAppName + ".app", "Contents");
```

Comment:
The changes proposed by TSRBerry close #5827.

I would also suggest putting the shortcut in the Applications folder on macOS.
```suggestion
            string contentFolderPath = Path.Combine("/Applications", cleanedAppName + ".app", "Contents");
```

Reply:
@marysaka is this desired?